ttnn/cpp/ttnn/run_operation_inl.hpp

```diff
@@ -216,6 +216,12 @@ void launch_op(
                         if (!output_tensor || !local_tensor) {
                             continue;
                         }
+
+                        // The return type is vector<optional<Tensor>>, and this refers to the case where the i-th value is nullopt.
+                        if (output_tensor->tensor_attributes.use_count() != 0 && local_tensor->tensor_attributes.use_count() == 0) {
```

Comment:
Can you please clarify the logic behind this change and what you need to achieve here?

Reply:
Actually never mind, I reread the comment and got it.